packages/office-addin-sso/src/configure.ts

```diff
@@ -66,22 +86,22 @@ async function createNewApplication(ssoAppName: string, port: string, userJson:
             if (await isUserTenantAdmin(userJson)){
                 await grantAdminContent(applicationJson);
                 await setTenantReplyUrls(applicationJson);
-            } else {
-                console.log(chalk.yellow("You are not a tenant admin so you cannot grant admin consent for your application.  Contact your tenant admin to grant consent"));
             }
 
             // Create an application secret and add to the credential store
             const secret: string = await setApplicationSecret(applicationJson);
             console.log(chalk.green(`App secret is ${secret}`));
             addSecretToCredentialStore(ssoAppName, secret);
 
+            usageDataHelper.sendUsageDataSuccessEvent('createNewApplication');
             return applicationJson;
         } else {
+            usageDataHelper.sendUsageDataException('createNewApplication', 'application data returned undefined');
             console.log(chalk.red("Failed to register application"));
             return undefined;
         }
-
     } catch (err) {
+        usageDataHelper.sendUsageDataException("createNewApplication", err);
```

Comment:
Is it OK to send arbitrary err value back? Does this get sanitized?

Reply:
yes it does get sanitized by the office-addin-usage-data package